### ISSUE TITLE
Can't Delete (+All Blue Screen)

### ISSUE THREAD

**Issue by oldcoyote** [stolksdorf/homebrewery#438]
Pages are showing up blue on both sides so I can't edit the code to fix whatever might be causing it. And even if I could, I checked through the source code and can't find any open code that might be creating the issue. 

On top of that, now I can't delete any unpublished brews from my main page - the trash icon shows up but does nothing. Is there any other way to figure out what's caused it or to delete the faulty page?

### Additional Details

**Share Link** : 
<img width="1356" alt="screen shot 2017-07-06 at 5 12 48 pm" src="https://user-images.githubusercontent.com/29941946/27899757-e461d3e0-626e-11e7-8700-71d0962fe7dd.png">

or

**Brew code to reproduce** : <details><summary>Click to expand</summary><code><pre>

PASTE BREW CODE HERE

</pre></code></details>

**Comment by calculuschild** [comment]
Yes, this is a result of some invalid code in your brew, causing the display to crash. And your document is pretty big. I would recommend grabbing your source code by clicking "share" and then "source". Copy your code into a new brew, but do a binary search by pasting the first half. Save, and if it works, the error must have been in the second half. Now paste in half of that. It broke? The error is somewhere in there. Paste half of that.

You should find the bad spot pretty easy.

As for deleting, are you saying you can't delete it from your profile page?

**Comment by oldcoyote** [comment]
That's correct. We've already established (via the same method) that the error is at the bottom somewhere. We've saved the data, we're not worried about rescuing this particular file - but when I try to delete it, there is no link on the trash can. The 'share' and 'edit' icons have links, the trash can does not - this applies to every one of my homebrews now, not just the one I actually want to delete.
